web/api/v2/minikit/app-metadata/[app_id]/index.ts

```diff
@@ -91,9 +91,13 @@ export const GET = async (
 
   const client = await getAPIServiceGraphqlClient();
 
+  const headers = req.headers;
+  const locale = headers.get("x-accept-language") ?? "";
```

Comment:
@andy-t-wang Are you sure these header values have the same format we are using for locales? Just clarifying

Reply:
Confirming with android team